theano/sandbox/cuda/dnn_base.c

```diff
@@ -5,31 +5,44 @@ static cudnnHandle_t _handle = NULL;
 static int
 c_set_tensorNd(CudaNdarray *var, cudnnTensorDescriptor_t desc) {
 
-
   int dim = CudaNdarray_NDIM(var);
-  int strides[dim];
+  int *strides = (int *)malloc(dim * sizeof(int));
```

Comment:
I think it works. I'll make sure and change it this weekend when I'm back
home.

On Mon, 24 Aug 2015 01:17 Frédéric Bastien <notifications@github.com> wrote:

> In theano/sandbox/cuda/dnn_base.c
> <https://github.com/Theano/Theano/pull/3308#discussion_r37711872>:
>
> >    int dim = CudaNdarray_NDIM(var);
> > -  int strides[dim];
> > +  int *strides = (int *)malloc(dim * sizeof(int));
>
> Can we use calloc on windows? I think it is faster on linux.
>
> Otherwise, thanks.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Theano/Theano/pull/3308/files#r37711872>.
>